src/src/WebServer/Markup.h

```diff
@@ -59,7 +59,12 @@ void addSelector(const String& id,
                  int           selectedIndex,
                  boolean       reloadonchange,
                  bool          enabled,
-                 const String& classname);
+                 const String& classname
+                 #ifdef        ENABLE_TOOLTIPS
+                 ,
+                 const String& title = ""
```

Comment:
`= EMPTY_STRING` should work here.

Reply:
Ok, should I add variants that use `__FlashStringHelper` too, or would that add too much extra code that's hardly ever used?

Reply:
Replaced the default value `""` for `tooltip` with `EMPTY_STRING`